force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -176,11 +197,12 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String getCurrentDate() {
         // Get the todays date
-
+        Date tday = System.today();
         // Format the current date
-
+        String formatToday = tday.format();
         // Return the result
-        return null; // Replace null with the variable you used to store the result
+        System.debug(formatToday);
+        return formatToday; // Replace null with the variable you used to store the result
```

Comment:
Best practice is to remove System.Debug statements.   If we think about how our code evlautes  it helps us see if we need those variables. Our method returns a String, and calling .format() on Date returns a String. 

```
 public static String getCurrentDate() {
        return Date.today().format();
    }
    ```